framework/Models/Model.py

```diff
@@ -132,7 +132,7 @@ class cls.
   validateDict['Optimizer'][0]['class'       ] ='Optimizers'
   validateDict['Optimizer'][0]['required'    ] = False
   validateDict['Optimizer'][0]['multiplicity'] = 1
-  validateDict['Optimizer'][0]['type']         = ['SPSA','FiniteDifference','ConjugateGradient']
+  validateDict['Optimizer'][0]['type']         = ['SPSA','FiniteDifference','ConjugateGradient', 'SimulatedAnnealing']
```

Comment:
Huh, I never updated this, and nothing errored out. This should only by GradientDescent and SimulatedAnnealing.